db/migrate/20191009004932_create_bulk_downloads_table.rb

```diff
@@ -0,0 +1,18 @@
+class CreateBulkDownloadsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bulk_downloads do |t|
+      t.text "params_json", comment: "JSON of the params for this bulk download"
+      t.string "download_type", comment: "The type of bulk download"
```

Comment:
if we want to want to get really fancy, we could use rails enum types here, and for `status`. disclaimer: I have not used them before.

Reply:
My main concern, apart from us having never used it before, is that enums get stored as numbers in the database, which makes using the Rails console much more annoying. (you have to remember what "0" and "1" mean) Will defer for now.